app/models/preserved_copy.rb

```diff
@@ -10,7 +10,7 @@ class PreservedCopy < ApplicationRecord
     fixity_check_failed: 5
   }
 
-  DEFAULT_STATUS = statuses[:ok].freeze
+  DEFAULT_STATUS = 'ok'.freeze
```

Comment:
it was my feedback that suggested that I didn't like the plain strings, because they aren't clearly part of an enum when out of context.  This one is, of course, in context.   I also suggested we have 

OK_STATUS = whatever
INVALID_MOAB_STATUS = ...

as conveniences, and because I think it makes me feel better and purport that it might be a compromise John is ok with.   It makes the values clearly constants when used elsewhere, and you don't have to magically know the details of the model.

Reply:
yup, sorry, i didn't call out that bit of contention clearly at all in the issue or PR, but that was my understanding from @SaravShah.  @SaravShah and @tingulfsen seemed to have a preference for strings, and you seemed somewhat open to that, and i had a slight preference for that, and it seems more conventional from everything i've read, hence this PR.

but i agree with the discomfort about them not clearly being enums out of context, and am totally happy to do the constants thing you suggest here, and will push a follow on commit with that.